squiral/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '0.1.1'
```

Comment:
Well, it's already been 0.1.2
https://github.com/sadikkuzu/squiral/blob/main/squiral/__init__.py#L1